tests/test_document.py

```diff
@@ -0,0 +1,109 @@
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+        <http://example.com/documents/1> a foaf:Document;
+                dct:title   "Title 1"@en, "Tittel 1"@nb ;
+        .
+        """
+    g1 = Graph().parse(data=document.to_rdf(), format="turtle")
+    g2 = Graph().parse(data=src, format="turtle")
+
+    assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_document_as_bnode() -> None:
+    """It returns a title graph isomorphic to spec."""
+    """It returns an identifier graph isomorphic to spec."""
+
+    document = Document()
+    document.title = {"nb": "Tittel 1", "en": "Title 1"}
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+        [ a foaf:Document;
+                dct:title   "Title 1"@en, "Tittel 1"@nb ; ]
+        .
+        """
+    g1 = Graph().parse(data=document.to_rdf(), format="turtle")
+    g2 = Graph().parse(data=src, format="turtle")
+
+    assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_language() -> None:
+    """It returns an identifier graph isomorphic to spec."""
+    document = Document()
+    document.identifier = "http://example.com/documents/1"
+    document.language = "http://example.com/languages/1"
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+        <http://example.com/documents/1> a foaf:Document;
+                dct:language "http://example.com/languages/1"^^dct:LinguisticSystem
+        .
+        """
+    g1 = Graph().parse(data=document.to_rdf(), format="turtle")
+    g2 = Graph().parse(data=src, format="turtle")
+
+    assert_isomorphic(g1, g2)
+
+
+def test_to_graph_should_return_format() -> None:
+    """It returns an identifier graph isomorphic to spec."""
+    document = Document()
+    document.identifier = "http://example.com/documents/1"
+    document.format = "http://example.com/formats/1"
+
+    src = """
+        @prefix dct: <http://purl.org/dc/terms/> .
+        @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+        @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+        @prefix dcat: <http://www.w3.org/ns/dcat#> .
+        @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+        <http://example.com/documents/1> a foaf:Document;
+                dct:format "http://example.com/formats/1"^^dct:MediaType
```

Comment:
dct:format fins i Distribution-klassen, der ser testen slik ut:
```
    @prefix dct: <http://purl.org/dc/terms/> .
    @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
    @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
    @prefix dcat: <http://www.w3.org/ns/dcat#> .
    <http://example.com/distributions/1> a dcat:Distribution ;
        dct:format <https://www.iana.org/assignments/media-types/application/pdf> ;
        .
```
bruken av dct:format her bør vere lik.
Ref: https://github.com/Informasjonsforvaltning/datacatalogtordf/blob/5649fee5bdb9a52abf14963f5618759b6ab40355/tests/test_distribution.py#L461

Reply:
Ok, jeg skal endre dette. Men bør jeg droppe range '^^dct:MediaType'?

Reply:
Ja, for range til dct:format er dct:MediaTypeOrExtent. Men som sagt: eg meiner dct:format skal ut av dette biblioteket.

Reply:
Ok, da tar jeg det ut. :+1: 

Men i modelldcatno-spekken står det at dct:format skal ha range dct:MediaType:

https://informasjonsforvaltning.github.io/modelldcat-ap-no/#Dokument-format

Men dette er kanskje feil?

Reply:
Ja, det har vore litt surr her, for dcat-ap-no var feil på eit tidspunkt. dcat-ap-no er no korrekt: https://data.norge.no/specification/dcat-ap-no/#Distribusjon-format

Reply:
Men modelldcat-ap-no står "fritt" i å seie at range _der_ skal vere dct:MediaType. Men eg vil tru at modelldcat-ap-no også kjem til å endre.